graphqlapi/src/main/java/io/stargate/graphql/schema/DmlSchemaBuilder.java

```diff
@@ -304,16 +310,22 @@ private GraphQLFieldDefinition buildDelete(Table table) {
   private List<GraphQLInputObjectField> buildFilterInputFields(Table table) {
     List<GraphQLInputObjectField> fields = new ArrayList<>();
     for (Column column : table.columns()) {
-      fields.add(
-          GraphQLInputObjectField.newInputObjectField()
-              .name(nameMapping.getGraphqlName(table, column))
-              .type(fieldFilterInputTypes.get(column.type()))
-              .build());
+      String graphqlName = nameMapping.getGraphqlName(table, column);
+      if (graphqlName != null) {
+        try {
+          fields.add(
+              GraphQLInputObjectField.newInputObjectField()
+                  .name(graphqlName)
+                  .type(fieldFilterInputTypes.get(column.type()))
+                  .build());
+        } catch (Exception e) {
+          warnings.add(
+              String.format(
+                  "Could not create filter input type for column %s.%s, skipping (%s)",
```

Comment:
NIT: We could add a more clear notation for column+table: `"[...] for column %s in table %s [...]"`